from exceptions import BadRequestException


def utility():
    pass
